pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -111,37 +111,37 @@ fi
 
 if [[ "$isCpp" = 1 ]]; then
     if [[ "$cppInclude" = 1 ]]; then
-        NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE ${NIX_CXXSTDLIB_COMPILE-@default_cxx_stdlib_compile@}"
+        NIX_@infixSalt@_CFLAGS_COMPILE="$NIX_@infixSalt@_CFLAGS_COMPILE ${NIX_@infixSalt@_CXXSTDLIB_COMPILE-@default_cxx_stdlib_compile@}"
     fi
-    NIX_CFLAGS_LINK="$NIX_CFLAGS_LINK $NIX_CXXSTDLIB_LINK"
+    NIX_@infixSalt@_CFLAGS_LINK="$NIX_@infixSalt@_CFLAGS_LINK $NIX_@infixSalt@_CXXSTDLIB_LINK"
 fi
 
 LD=@ldPath@/ld
 source @out@/nix-support/add-hardening.sh
```

Comment:
Unless there is a reason to keep the name `infixSalt`, why not name the symbol `target_platform_with_dashed_replaced_by_underscores` (I have nothing against long identifiers, as you can see)?